src/components/panels/SpoolmanPanel.vue

```diff
@@ -0,0 +1,161 @@
+import SpoolmanEjectSpoolDialog from '@/components/dialogs/SpoolmanEjectSpoolDialog.vue'
+import { ServerSpoolmanStateSpool } from '@/store/server/spoolman/types'
+import SpoolmanPanelActiveSpool from '@/components/panels/Spoolman/SpoolmanPanelActiveSpool.vue'
+
+@Component({
+    components: { SpoolmanPanelActiveSpool, Panel, SpoolmanChangeSpoolDialog, SpoolmanEjectSpoolDialog },
+})
+export default class SpoolmanPanel extends Mixins(BaseMixin) {
+    mdiAdjust = mdiAdjust
+    mdiDotsVertical = mdiDotsVertical
+    mdiEject = mdiEject
+    mdiOpenInNew = mdiOpenInNew
+    mdiSwapVertical = mdiSwapVertical
+
+    showChangeSpoolDialog = false
+    showEjectSpoolDialog = false
+
+    get health() {
+        return this.$store.state.server.spoolman.health ?? ''
+    }
+
+    get title() {
+        const headline = this.$t('Panels.SpoolmanPanel.Headline') as string
+
+        if (this.health === '' || this.health === 'healthy') return headline
+
+        return `${headline} (${this.health})`
+    }
+
+    get changeSpoolTooltip(): string {
+        if (this.active_spool === null) return this.$t('Panels.SpoolmanPanel.SelectSpool') as string
+
+        return this.$t('Panels.SpoolmanPanel.ChangeSpool') as string
+    }
+
+    get active_spool(): ServerSpoolmanStateSpool | null {
+        return this.$store.state.server.spoolman.active_spool ?? null
+    }
+
+    get color() {
+        const color = this.active_spool?.filament.color_hex ?? null
+        if (color === null) return '#000'
+
+        return `#${color}`
+    }
+
+    get id() {
+        return this.active_spool?.id ?? 'XX'
+    }
+
+    get vendor() {
+        return this.active_spool?.filament?.vendor?.name ?? 'Unknown'
+    }
+
+    get name() {
+        return this.active_spool?.filament.name ?? 'Unknown'
+    }
+
+    get materialOutput() {
+        const material = this.active_spool?.filament.material ?? null
+        if (material === null) return null
+
+        return material
+    }
+
+    get weightOutput() {
+        let remaining = this.active_spool?.remaining_weight ?? null
+        let total = this.active_spool?.filament.weight ?? null
+        let unit = 'g'
+
+        if (remaining === null || total === null) return null
+        remaining = Math.round(remaining)
+        let totalRound = Math.floor(total / 1000)
+
+        if (total >= 1000) {
+            if (totalRound !== total / 1000) {
+                totalRound = Math.round(total / 100) / 10
+            }
+
+            return `${remaining}g / ${totalRound}kg`
+        }
+
+        return `${remaining} / ${total}${unit}`
+    }
+
+    get lengthOutput() {
+        let remaining = this.active_spool?.remaining_length ?? null
+
+        if (remaining === null) return null
+        remaining = Math.round(remaining / 1000)
+
+        return `${remaining}m`
+    }
+
+    get subtitle() {
+        return [this.materialOutput, this.weightOutput, this.lengthOutput].filter((v) => v !== null).join(' | ')
+    }
```

Comment:
Unsued getters... are these requried here? -> Duplicates in `SpoolmanPanelActiveSpool.vue`

Reply:
ohh thanks! i forgot to remove them...